src/parser.py

```
@@ -347,17 +353,11 @@ def solr_post_json_data(self, spjd_base):
                                                                     base_name = spjd_base,
                                                                     logger_name = self.name
                                                                 )
-                # шлём, пока не пройдёт --------------------------------------------------------------------------------
-                while spjd_ret_code                         !=  200 and not self.stopMe:                                 # http://localhost:8983/solr/PER/update?wt=json
-                    t.debug_print(f"Post data returned {str(spjd_ret_code)}, retrying")
+                if spjd_ret_code                            !=  200:
+                    t.debug_print(f"Post data returned {str(spjd_ret_code)}, retrying", self.name)
```

Ruff (0.15.15)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)